test/league_stats_test.rb

```diff
@@ -0,0 +1,166 @@
+    game_11_info = {:game_id=>"2014020548", :season=>"20142015", :type=>"R", :date_time=>"2015-01-01", :away_team_id=>"12", :home_team_id=>"5", :away_goals=>"1", :home_goals=>"7", :outcome=>"home win OT", :home_rink_side_start=>"right", :venue=>"MTS Centre", :venue_link=>"/api/v1/venues/	null", :venue_time_zone_id=>"America/Winnipeg", :venue_time_zone_offset=>"-5", :venue_time_zone_tz=>"CDT"}
+    game_12_info = {:game_id=>"2014020016", :season=>"20142015", :type=>"R", :date_time=>"2014-10-10", :away_team_id=>"52", :home_team_id=>"53", :away_goals=>"5", :home_goals=>"1", :outcome=>"home win OT", :home_rink_side_start=>"right", :venue=>"MTS Centre", :venue_link=>"/api/v1/venues/	null", :venue_time_zone_id=>"America/Winnipeg", :venue_time_zone_offset=>"-5", :venue_time_zone_tz=>"CDT"}
+    game_13_info = {:game_id=>"test", :season=>"20142015", :type=>"R", :date_time=>"2014-10-10", :away_team_id=>"53", :home_team_id=>"52", :away_goals=>"1", :home_goals=>"0", :outcome=>"home win OT", :home_rink_side_start=>"right", :venue=>"MTS Centre", :venue_link=>"/api/v1/venues/	null", :venue_time_zone_id=>"America/Winnipeg", :venue_time_zone_offset=>"-5", :venue_time_zone_tz=>"CDT"}
+
+    @game_1 = Game.new(game_1_info)
+    @game_2 = Game.new(game_2_info)
+    @game_3 = Game.new(game_3_info)
+    @game_4 = Game.new(game_4_info)
+    @game_5 = Game.new(game_5_info)
+    @game_6 = Game.new(game_6_info)
+    @game_7 = Game.new(game_7_info)
+    @game_8 = Game.new(game_8_info)
+    @game_9 = Game.new(game_9_info)
+    @game_10 = Game.new(game_10_info)
+    @game_11 = Game.new(game_11_info)
+    @game_12 = Game.new(game_12_info)
+    @game_13 = Game.new(game_13_info)
+
+    @games = [
+              @game_1,
+              @game_2,
+              @game_3,
+              @game_4,
+              @game_5,
+              @game_6,
+              @game_7,
+              @game_8,
+              @game_9,
+              @game_10,
+              @game_11,
+              @game_12,
+              @game_13
+             ]
+
+    @league_stats = LeagueStats.new(@games, @teams)
+    @league_stats.sort_total_league_games
+  end
+
+  def test_it_exists
+    assert_instance_of LeagueStats, @league_stats
+  end
+
+  def test_it_has_games
+    assert_equal @games, @league_stats.games
+  end
+
+  def test_it_has_teams
+    assert_equal @teams, @league_stats.teams
+  end
+
+  def test_it_sorts_team_league_games
+    assert_equal 2, @team_6.home_wins_league
+    assert_equal 1, @team_6.away_wins_league
+    assert_equal 2, @team_6.away_goals_league
+    assert_equal 8, @team_6.home_goals_league
+    assert_equal 2, @team_3.away_games_league
+    assert_equal 1, @team_3.home_games_league
+    assert_equal 3, @team_6.total_wins_league
+    assert_equal 3, @team_3.total_games_league
+    assert_equal 5, @team_3.total_goals_league
+    assert_equal 10, @team_3.total_goals_allowed_league
```

Comment:
Is there a reason we aren't testing both teams the same way? Team 6 is being run for total wins but team 3 isn't and vice versa for total goals allowed.